docs/email/MX-Backup.md

```diff
@@ -0,0 +1,161 @@
+	libunicode-string-perl \ 
+	liburi-perl
+
+After that run 
+
+	`cpan Data:Uniqid`
+	
+Now you can run the smoke test. If it succeeds it will print some useful information about the imapsync version you are using and then the help page just as if you used `imapsync`.
+
+	`./imapsync`
+
+If all is working fine copy it into the right directory so that it is accessible from everywhere 
+
+	`cp imapsync /usr/bin/`
+	
+Test your installation of imapsync by typing `imapsync` in order to get a full printout of the features it offers (there are a lot of features).
+
+
+
+## Automate your backup
+If you have read through the feature list you might have noticed that   
+a. there are quite some information you have to enter in order to get it to work   
+and  
+b. it doesn't say anything about repetitive syncing or automation. So how does this work?
+
+The answer to this is quite easy: Bash scripts and Cron jobs. That might be a little "primitive" but truth be told both are very reliable and easy to setup. If you do not already know about Cron you should take a look at the intro guide in the [Linode library](/docs/tools-reference/tools/schedule-tasks-with-cron).
+
+All Cron does is basically wake up every minute, check all tasks, see if they apply to the current minute in time and if so it runs them.
+
+### Writing the script
+
+First we need to write our little shell script. In it we will only put the imapsync command which will sync the emails and the prints all of it into a single log file and after that enters a couple of line breaks and info into the given log file.
+
+{: .caution}
+>
+> You might have to create the imapsync folder under `/var/log` first using `mkdir /var/log/imapsync`. Otherwise you will not be able to get the logging working.
+
+{: .file}
+~/cron-imapsync.sh 
+:	~~~ bash
+	#! /bin/bash
+	
+	# This will sync your first email account automatically.
+	
+	imapsync --host1 youremailprovider.com --user1 youruser --password1 yourpassword --tls1 \
+	--host2 backup.yourdomain.com --user2 yourbackupuser --password2 yourbackupuserpassword --tls2 \
+	--nolog 2>&1 | tee -a /var/log/imapsync/imapsync.log
+	
+	echo "\n \n ///// \n \n End of the first transfer \n \n ///// \n \n" >> /var/log/imapsync/imapsync.log
+	
+	
+	# Copy and paste the above block as many times as you need to and replace all the values for the email servers (--host1 / --host2), users (--user1 / --user2) and password flags (--password1 / --password2).
+	~~~
+
+After that we need to make the script executable with `chmod -x ~/cron-imapsync.sh`. You can move the script wherever you feel like but it needs to be accessible by cron.
+
+### Cron
+
+Setting up Cron is actually quite simple. Open the editor with  
+	`crontab -e`  
+if you have never opened Cron before it will prompt you to pick the editor you want to use. Pick the one you prefer and then add the following at the end of the file.
+
+{: .file-excerpt}
+:	~~~
+	* 0 * * * sh /path-to-where-you-put-the-script
```

Comment:
There is an error here. Should be "00 0 * * * sh ...."

Otherwise it will execute 59 times after midnight